.github/workflows/test.yml

```diff
@@ -37,8 +44,6 @@ jobs:
           - 5432:5432
     steps:
       - uses: actions/checkout@v2
-        with:
```

Comment:
We don't need this?

Reply:
We need it for Sonar, but not for tests—since we kicked sonar to a separate job, this job can use the cheap checkout.